src/app/(with-navbar)/page.tsx

```diff
@@ -13,7 +13,7 @@ const Home = () => (
       <NewsletterForm />
       <FeaturePreview
         cta={{
-          href: "/components",
+          href: "/docs#components",
```

Comment:
How about changing the Greeting/Hero's CTA to Get Started and link that to just the `/docs` page?

The first CTA would say "Get Started" direct to `/docs`
Then the components CTA would say "Explore all components" or just "Explore components" and direct to the `/docs#components`